README.md

```diff
@@ -380,6 +386,101 @@ const vttOutput = webvtt(result);
 
 [See our standalone captions library for more information](https://github.com/deepgram/deepgram-node-captions).
 
+## Voice Agent
+
+```js
+import { createClient } from "@deepgram/sdk";
+import { AgentEvents } from "@deepgram/sdk";
+
+const deepgram = createClient(DEEPGRAM_API_KEY);
+
+// Create an agent connection
+const agent = deepgram.agent.live();
+
+// Set up event handlers
+agent.on(AgentEvents.Open, () => {
+  console.log("Connection opened");
+
+  // Configure the agent once connection is established
+  agent.configure({
+    audio: {
+      input: {
+        encoding: "linear16",
+        sampleRate: 16000,
+      },
+      output: {
+        encoding: "linear16",
+        container: "wav",
+        sampleRate: 24000,
+      },
+    },
+    agent: {
+      listen: {
+        model: "nova-3",
+      },
+      speak: {
+        model: "aura-asteria-en",
+      },
+      think: {
+        provider: {
+          type: "anthropic",
+        },
+        model: "claude-3-haiku-20240307",
+        instructions: "You are a helpful AI assistant. Keep responses brief and friendly.",
+      },
+    },
+  });
+});
+
+// Handle agent responses
+agent.on(AgentEvents.Speech, (speech) => {
+  console.log("Agent is speaking:", speech);
+});
+
+agent.on(AgentEvents.Transcript, (transcript) => {
+  console.log("User said:", transcript);
+});
```

Comment:
These are not accurate. The enum values are not correct, and the payloads received in these events may not be what you're thinking they are.

`AgentStartedSpeaking` sends an object with latency information.

```suggestion
agent.on(AgentEvents.AgentStartedSpeaking, (data) => {
  console.log("Agent started speaking:", data["total_latency"]);
});

agent.on(AgentEvents.ConversationText, (message) => {
  console.log(`${message.role} said: ${message.content}`);
});
```

Reply:
@naomi-lgbt  thank you!